airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -459,7 +459,7 @@ def _handle_job_error(job: BigQueryJob | UnknownJob) -> None:
             raise AirflowException(f"BigQuery job {job.job_id} failed: {job.error_result}")
 
     def execute_complete(self, context: Context, event: dict[str, Any]) -> None:
-        """Callback for when the trigger fires.
+        """Act as a callback for when the trigger fires.
```

Comment:
was not sure if I should go with `Act as a callback for when the trigger fires.` or `Act as a callback when the trigger fires.`

Reply:
Works for me.